### DISCUSSION TITLE
Why so many releases?

### DISCUSSION THREAD

**Discussion by dimaischenko** [hyochan/react-native-iap#1821] [General]
There have been a lot of releases lately (8-th version), sometimes several times a day. The library is perfect and cool and I have no complaints, it's just interesting to understand what is the logic of such frequent releases?

**Comment by hyochan** [comment]
Hello! This is a great question! A few months ago, there has been a new maintainer @andresesfm who is driving the repository ambitiously. He did lots of great things recently and migrates the package to recent updates like [Storekit2](https://developer.apple.com/storekit) and [Billing Library v5](https://developer.android.com/google/play/billing/migrate-gpblv5). Since then, we had to refactor the old codebases to make the package compatible and suffered from lots of breaking changes. We should all thank @andresesfm for the great work 👍.

Also, there is a newcomer @jeremybarbet who is passionately driving the package together. Therefore, the package these days is moving quickly eliminating past errors and changing lots of codebases. 

`BillingClient` is very time-consuming engineering work since there are lots of invisible things on the fly. I'd like to suggest people to [fund the project](https://opencollective.com/react-native-iap#sponsor) so that maintainers can grab some reward for their hard work. We are also funding today and I've not gotten any on working this so far. I wish the next maintainers do not just volunteer service 🙏

**Reply by dimaischenko** [reply]
Got it! @hyochan thank you for clarification. And thank you guys for such a cool and necessary project. I will contribute to the fund 💰

**Reply by hyochan** [reply]
Thank you for your warmness. The maintainers will be delighted ☺️
